quickstack/manifests/compute_common.pp

```diff
@@ -71,7 +71,10 @@
   $nova_uuid                    = $quickstack::params::nova_uuid,
   $rbd_key                      = $quickstack::params::rbd_key,
   $ceph_vlan                    = $quickstack::params::ceph_vlan,
-
+  $sensu_rabbitmq_host          = $moc_config::params::sensu_rabbitmq_host,
```

Comment:
I think we need to use quickstack::params:sensu_rabbitmq_host here. The same holds good for next two lines as well